ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -769,6 +791,49 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{name}-{test_name}-for-period':
+          test_name: windows-e2e-userspace
+          node: 'antrea-windows-testbed'
+          description: 'This is for running e2e tests with OVS userspace mode on a testbed.'
```

Comment:
What do you mean by "OVS userspace mode" here? For me, it means the OVS userspace datapath (as opposed to the kernel datapath). I am not sure this is what you mean here.

Reply:
yes it means the same.

Reply:
It doesn't look that way to me. You seem to use it as a toggle to differentiate between "OVS userspace daemons are running as a container" and "OVS userspace daemons are running as Windows services".
This is very different from what I'm describing (which is this: https://docs.openvswitch.org/en/latest/intro/install/userspace/), unless I am missing something.
If I am correct the naming is very confusing, and so is the description here.

Reply:
I have updated the description of the job, basically in this job we are running OVS userspace daemons as windows services, instead of running them inside container.